src/modules/ROOT/pages/changelog/202303/01.json

```diff
@@ -0,0 +1,889 @@
+        "https://github.com/cortezaproject/corteza/pull/524/commits/950d305a5a8fb79ee84599989a399af7d028f599"
+      ]
+    },
+    {
+      "what": [
+        "the ability to copy, clone, and paste page blocks in page builders."
+      ],
+      "why": [
+        "to make it possible to duplicate page blocks easily."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/582/commits/7f377109d0b46ade1b04857569622829f917321b"
+      ]
+    },
+    {
+      "what": [
+        "an option to magnify certain page blocks when viewing a public Compose page.",
+        "A select dropdown has been added in the general tab of page block configuration modal for calendar, chart, content, comments, file, iframe, record list, report and record revisions page blocks.",
+        "The user can select between Fullwidth or Modal select option, and when enabled, the user will see a magnify icon at the top of the block."
+      ],
+      "why": [
+        "to help users finding the info they are looking for much faster and easier with the option to magnify certain page blocks."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/576/commits/1a836ddf2c0c9e5892b791ba201032d5d8a37365"
+      ]
+    },
+    {
+      "what": [
+        "a spinner to indicate when an action in template editor is submitted."
+      ],
+      "why": [
+        "to have an UI consistency, because it was difficult to tell when your changes were being submitted."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/708/commits/7a4a0dfc87756194f66365f1aad26b1447b4b2b5"
+      ]
+    },
+    {
+      "what": [
+        "a module fields description in the Module List configurator that reads: 'If no fields are selected, the default values would be shown.'"
+      ],
+      "why": [
+        "to have a better user experience."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/751/commits/b4d5c7fd7cdef17c847717e2944fc669c213204d"
+      ]
+    },
+    {
+      "what": [
+        "an empty table with `No record revisions found` message in admin record template.",
+        "Added the message in the public record page, when there are no revisions."
+      ],
+      "why": [
+        "to have a better user experience when there are no record revisions present."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/758/commits/2357851caa1b16a43b27fdfc379cfd92c7dcef7b"
+      ]
+    },
+    {
+      "what": [
+        "manual refresh option so that users can choose when to refresh a block."
+      ],
+      "why": [
+        "to give users more control over refreshing when auto-refreshing is not needed."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/760/commits/954419bac6c3d19e3c5c46adf93d908528ad18df"
+      ]
+    },
```

Comment:
Let's have this merged with the auto-refresh one.
Generally, when there are additions/modifications to new features (like the PB refresh one), I merge them all together.